doc/underpinnings.tex

```diff
@@ -0,0 +1,779 @@
+\documentclass[margin=5mm]{article}
+\usepackage{tikz}
+\usepackage{parskip}
+\usepackage{listings}
+\usepackage{color}
+\usepackage{amsmath}
+
+
+\usepackage[utf8]{inputenc}
+\usepackage[english]{babel}
+ \usepackage{hyperref}
+\hypersetup{
+    colorlinks=true,
+    linkcolor=blue,
+    filecolor=magenta,      
+    urlcolor=cyan,
+}
+ 
+\urlstyle{same}
+
+\usetikzlibrary{matrix,calc}
+\definecolor{dkgreen}{rgb}{0,0.6,0}
+\definecolor{gray}{rgb}{0.5,0.5,0.5}
+\definecolor{mauve}{rgb}{0.58,0,0.82}
+
+\lstset{frame=tb,
+  aboveskip=3mm,
+  belowskip=3mm,
+  showstringspaces=false,
+  columns=flexible,
+  basicstyle={\small\ttfamily},
+  numbers=none,
+  numberstyle=\tiny\color{gray},
+  keywordstyle=\color{blue},
+  commentstyle=\color{dkgreen},
+  stringstyle=\color{mauve},
+  breaklines=true,
+  breakatwhitespace=true,
+  tabsize=3
+}
+
+
+\begin{document}
+
+\section{Send Population Model}
+
+From Simulation, Modeling and Analysis:
+
+\begin{quote}
+The state of a system is defined to be the collection of variables
+necessary to describe a system at a particular point in time, relative
+to the objective of a study.
+\end{quote}
+
+and
+
+\begin{quote}
+A system is defined to be the collection of entities that act and
+interact towards the accomplishment of some goal.
+\end{quote}
+
+In the Send Model the collection of entities that make up the system
+are drawn from $S \times N \times AY$ where $S = \{MMSIB, ISS, ISSR, \dots, MU\}$,
```

Comment:
Clarify what these represent. E.g. add sentence: "For a given local authority, S represents the set of valid types of school setting, N represents the set of valid need types, and AY is the set of allowable academic years in SEND."